Makefile

```diff
@@ -1,21 +1,18 @@
-BIN := venv/bin
-PYTHON := $(BIN)/python
 DEV_LISTEN_IP := 0.0.0.0
 DEV_PORT := 3000
 
 .PHONY: dev
-dev: venv
-	@echo -e "\e[1m\e[93mRunning on http://$(shell hostname -f):$(DEV_PORT)/\e[0m"
-	$(PYTHON) ./manage.py runserver $(DEV_LISTEN_IP):$(DEV_PORT)
+dev: venv serve
 
-venv: requirements.txt requirements-dev.txt
-	python ./vendor/venv-update venv= venv -ppython3 install= \
-		-r requirements.txt -r requirements-dev.txt
+venv: Pipfile Pipfile.lock
+	pipenv install --dev
+
+serve:
+	@echo -e "\e[1m\e[93mRunning on http://localhost:$(DEV_PORT)/\e[0m"
+	pipenv run python ./manage.py runserver $(DEV_LISTEN_IP):$(DEV_PORT)
 
-.PHONY: clean
 clean:
-	rm -rf venv
+	pipenv uninstall --all --rm
```

Comment:
I couldn't get this to work, pipenv said `Error: no such option: --rm` unless I just made this into `pipenv --rm`, which worked fine. Looking at `pipenv --help`'s output, `--rm` says that it completely removes the virtualenv, while `uninstall --all` just removes all packages. If the goal is to just remove everything, wouldn't just using `pipenv --rm` work since it removes the whole virtualenv?

Reply:
Yeah, probably best to just remove everything with --rm.